infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/InLogEntrySyncState.java

```diff
@@ -148,8 +148,7 @@ public void onEntry(LogReplicationState from) {
                 fsm.getAckReader().markSnapshotSyncInfoCompleted();
             }
 
-            logEntrySyncFuture = fsm.getLogReplicationFSMWorkers().submit(() ->
-                    logEntrySender.send(transitionEventId));
+            logEntrySender.send(transitionEventId);
```

Comment:
Yes. Earlier we had a different threadpool called worker threads. These did the actual work of reading the logs and sending to remote and waiting for ack.
The reason for this to be async seemed to be so that the consumer threads that processed the events would not be blocked.
Now that there is only 1 threadpool, there is no need for this "work" to be scheduled/or be done asynchronously.

Reply:
ok, got it.  Now the workers and consumers share the same pool and it size is 2 (instead of 1 each).  Thanks.